src/Button.js

```diff
@@ -47,7 +48,7 @@ class Button extends React.Component {
     return (
       <Component
         {...elementProps}
-        type={elementProps.type || 'button'}
```

Comment:
slightly different semantically... do we ever pass in falsy (but defined) `type` here?